{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -260,7 +260,7 @@
 
 PRIVATE_MEDIAFILES_LOCATION = ""
 # Django Storages configuration
-if config("USE_AWS_STORAGE", cast=bool, default=False) and not IS_REVIEW_APP:
+if config("USE_AWS_STORAGE", cast=bool, default=False):
```

Comment:
Not necessarily opposed to the removal, but a couple of things:

- See https://github.com/thinknimble/tn-spa-bootstrapper/pull/138 as @paribaker added a reason for this change being made
- If we are going to remove this, then perhaps we consider removing `IS_REVIEW_APP` and `IN_REVIEW` as well, since they have very limited use

Reply:
well the reason it is in there is so that we do not by default store things and instead explicitly do it (but I guess you can also just not provide the .env variables). The reason for this is because when the review app is destroyed we do not have a process for clearing out the assets

Reply:
yea, and I know some have run into this "annoying" extra safe guard we added when trying to do something specific in a review app. Just want to make sure Elias is thinking of the implications and any broader changes.

I personally think both `IS_REVIEW_APP` and `IN_REVIEW` should just be removed all together. But I'm also fairly indifferent on this as well.

Reply:
If it becomes an issue we can re-add it ! Approved for a merge IMO! @eliasbiagioninc